tests/vector_stores/test_elasticsearch.py

```diff
@@ -28,7 +37,9 @@
     elasticsearch_not_available = False
 
     es_license = es_client.license.get()
-    basic_license: bool = "basic" == es_license["license"]["type"]
+    # TODO: Fix hybrid search tests
```

Comment:
ATM tests that require RRF are off due to bug. see issue for more info https://github.com/jerryjliu/llama_index/issues/7830